hpy/devel/include/universal/autogen_trampolines.h

```diff
@@ -236,6 +240,14 @@ static inline int HPy_SetItem_s(HPyContext ctx, HPy obj, const char *key, HPy va
      return ctx->ctx_SetItem_s ( ctx, obj, key, value ); 
 }
 
+static inline void *_HPy_Cast(HPyContext ctx, HPy h) {
+     return ctx->ctx_Cast ( ctx, h ); 
+}
+
+static inline HPy HPy_New(HPyContext ctx, HPy h_type, void **data) {
+     return ctx->ctx_New ( ctx, h_type, data ); 
```

Comment:
A pure performance hack... Writing it this way is probably better for callers:

`void *data_result; HPy h = ctx->ctx_New(ctx, h_type, &data_result); *data = *data_result; return h;`

Without it, the C compiler must assume that if the caller passes, say, `&point_data` then anybody could write a different value at any time into this local variable because a pointer to it escaped.  With this hack, it's no longer the case: what escaped is the address of `data_result` instead and that local variable disappears.

Reply:
interesting, I'd have never thought about that.
Fixed by 1396a4a.